odnoklassniki-android-sdk/src/main/java/ru/ok/android/sdk/Odnoklassniki.java

```diff
@@ -409,10 +437,41 @@ private void signParameters(final Map<String, String> params) {
         params.put(Shared.PARAM_SIGN, sig);
     }
 
-    public final void setOkListener(OkListener listener) {
+    public final void setOkListener(final OkListener listener) {
```

Comment:
@SKART1 Yes, this description should lie in javadoc telling why listener will work the way you proposed, and what results will the developer achieve.